RELEASING.md

```diff
@@ -1,48 +1,63 @@
-# Releasing OpenTelemetry Packages (for maintainers only)
-This document explains how to publish all OT modules at version x.y.z. Ensure that you’re following semver when choosing a version number.
+# Release instructions
+
+## Preparing a new major or minor release
+
+* Merge a pull request to `main` updating the `CHANGELOG.md`.
+  * The heading for the release should include the release version but not the release date, e.g.
+    `## Version 1.9.0 (unreleased)`.
+  * Use `.github/scripts/draft-change-log-entries.sh` as a starting point for writing the change
+    log.
+* Run the [Prepare release branch workflow](https://github.com/trask/repository-template/actions/workflows/prepare-release-branch.yml).
```

Comment:
Should there be a step before this that informs to update `eachdist.ini` to the correct versions?

Reply:
The versions in `eachdist.ini` are automatically updated as part of the workflows

Reply:
Could you refer to the line in which this happens?

Reply:
ya, https://github.com/open-telemetry/opentelemetry-python/pull/2649/files#diff-629de75a59c87199cb26eafcb3067377c3b88c4cbb9be6b7cace2d0d68f41183R31